### PR TITLE
Local Date-Time, Local Date, Local Time

### PR DESCRIPTION
**Pull request by KitaitiMakoto**
Hi,

Thank you for fast and dependencyless TOML parser!

As said in #20, I added that classes and fixed parser to use them. Can you check patches?

### CODE REVIEW

**Comment by fbernier** [comment]
Hey, thanks for all the patches and sorry for not replying earlier. I can't put much more time on this as you saw but if you want I would gladly give you commit access so you can merge all of this and future ones. I'll still try to review your PRs if I can.

File: lib/tomlrb/local_date_time.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
+module Tomlrb
+  class LocalDateTime
+    def initialize(year, month, day, hour, min, sec)
+      @time = Time.new(year, month, day, hour, min, sec, '-00:00')
+      @sec = sec
+    end
+
+    def year
```
Comment (fbernier) [inline lib/tomlrb/local_date_time.rb]: I think we could do
```ruby
def_delegators :@time, :year, :month, :day, :hour, :min, :sec, :usec, :nsec
```

and avoid quite a lot of repetition in all these time classes

**Comment by KitaitiMakoto** [comment]
@fbernier Thank you for reply.

> but if you want I would gladly give you commit access so you can merge all of this and future ones.

It's so appreciated if you do so. Thanks!

File: lib/tomlrb/local_date_time.rb (hunk 1 above)
Comment (KitaitiMakoto) [reply lib/tomlrb/local_date_time.rb]: Okay, I will rewrite.

Comment (KitaitiMakoto) [reply lib/tomlrb/local_date_time.rb]: Done.

**Pull request merged**